apps/web/cypress/e2e/single-dwelling/single-dwelling-9.9.9-results.cy.ts

```diff
@@ -23,6 +24,8 @@ describe("single dwelling: 9.9.9 walkthrough results", () => {
   });
 
   it("should display correct ui elements", () => {
+    cy.getByTestID(TESTID_BREADCRUMBS).should("be.visible");
```

Comment:
Adding check for the breadcrumbs as a whole exist. we test each individual section once in the multi-dwelling-walkthrough